tests/testthat/test-error.R

```diff
@@ -242,6 +242,22 @@ test_that("errors in r_session$call() are merged", {
   }
 })
 
+test_that("early error for missing environment file", {
+
+  f <- function() {
+    tmp <- tempfile()
+    file.rename(callr:::env_file, tmp)
+    on.exit(file.rename(tmp, callr:::env_file))
```

Comment:
This runs in a throwaway session, so you don't need to restore the file. I suggest just to remove it, that expresses the intent better, I think.

Reply:
Actually, it is fine, no worries.